extensions/wikia/WikiaApi/WikiaApiQueryAllUsers.php

```diff
@@ -320,23 +308,23 @@ private function local_users() {
 				}
 
 				// No more rows left
-				if (!$row) break;
+				if ( !$row ) break;
 
-				if ($count > $limit) {
+				if ( $count > $limit ) {
 					$this->setContinueEnumParameter( 'from', $this->keyToTitle( $row->user_name ) );
 					break;
 				}
 
 				// Record new user's data
 				$lastUser = $row->user_name;
 				$lastUserData = array( 'name' => $lastUser );
-				
+
 				# is blocked
-				if ($this->fld_blockinfo) {
+				if ( $this->fld_blockinfo ) {
 					$blocker_name = $block_reason = '';
 					if ( $row->user_is_blocked ) {
-						$oBlock = Block::newFromTarget( User::whoIs($row->user_id), 0 ); 
-						if ( is_object($oBlock) ) {
+						$oBlock = Block::newFromTarget( User::whoIs( $row->user_id ), 0 );
+						if ( is_object( $oBlock ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)